xarray/plot/plot.py

```diff
@@ -449,10 +449,28 @@ def newplotfunc(darray, x=None, y=None, figsize=None, size=None,
         if imshow_rgb:
             # Don't add a colorbar when showing an image with explicit colors
             add_colorbar = False
+            # Calculate vmin and vmax automatically for `robust=True`
+            if robust:
+                if vmin is None:
+                    vmin = np.nanpercentile(darray, ROBUST_PERCENTILE)
+                if vmax is None:
+                    vmax = np.nanpercentile(darray, 100 - ROBUST_PERCENTILE)
+                robust = False
+            # Scale interval [vmin .. vmax] to [0 .. 1] and clip to bounds
+            if vmin is not None or vmax is not None:
+                vmin = vmin if vmin is not None else darray.min()
+                vmax = vmax if vmax is not None else darray.max()
```

Comment:
Shouldn't the default here for vmin/vmax be 0/1 rather than the min/max values of the array? That would seem a little less surprising to me. Otherwise setting one of `vmin`/`vmax` would cause the other to be automatically set which seems strange.